packages/components/addon/components/hds/form/error/message.hbs

```diff
@@ -0,0 +1,3 @@
+<p class="hds-form-error__message" ...attributes>
+  {{yield}}
+</p>
```

Comment:
[a11y improvement suggestion] [equal experience] Looking through the docs - since we're supporting multiple error messages, it would be an improvement to put these inside of a `ul`/`li` structure. The reason is that the list count is read out to the user with assistive tech, so they have a good idea of how many errors there are, without manually counting. This allows the user to have a more complete idea of what is going on, and provides them with the same information that users already know if they can visually consume the information on the screen.

The current implementation does not violate any current WCAG Success Criteria.

Reply:
@alex-ju can chime in on this about his thinking in more detail. What I can say is that we discussed this specifically, but since there's no way in Ember to know which (and so how many) contextual components are passed as children, we can't discern when there's a single error message vs when there are multiple ones. We came to the conclusion that since in most of the cases we have a single message, it would be weird (and also maybe even less accessible) to have a list that contains one single message item